### ISSUE TITLE
A11y_ML .NET Model Builder_ Environment_ Screen Reader: Narrator/NVDA focus is not going to "Environment" information and to the extra content inside each environment tab item in scan mode

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1129]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev1; #Benchmark; #Visual Studio; #Screen Reader; #Narrator; #NVDA; #WCAG1.3.2;

### Environment Details:
Microsoft Visual Studio 2019preview Version 16.8.0 Preview 6.0

### Screen Readers:
Narrator; NVDA

### Note:
Same issue is observed in "Environment" tab items of all the scenarios

### Repro Steps:

1.Start Narrator
2.Open Visual Studio Enterprise 2019 Preview 
3. Create a new .Net Core Console App (C#) 
4. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
5. Tab till "Text Classification" scenario and select the scenario
6. Enable narrator scan mode and use up/down arrows (or Caps lock + up/down arrows) to navigate to environment details on the left side and verify whether user is able to do so

### Actual Result:
Narrator/NVDA focus is not going to "Environment" information in scan mode. Also narrator focus is not going to the extra content inside each environment tab item

### Expected Result:
Screen reader users should be able to navigate to non actionable content using narrator scan mode or NVDA browse mode (Insert + Numpad arrow keys)

### User Impact:
Screen Reader users will miss the content if user is not able to navigate to non actionable content in narrator scan mode

### Attachment for Reference

[1129_ML .NET Model Builder_ Narrator is not reading non interactive content.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5629610/1129_ML.NET.Model.Builder_.Narrator.is.not.reading.non.interactive.content.pptx)
[1129_ML .NET_Img Class_Narrator focus is not going to extra content.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5671046/1129_ML.NET_Img.Class_Narrator.focus.is.not.going.to.extra.content.pptx)

**Comment by SaisreeDandamudi** [comment]
[1129_ML.NET.Model.Builder_Fixed_Narrator.is.not.reading.non.interactive.content.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/6042236/1129_ML.NET.Model.Builder_Fixed_Narrator.is.not.reading.non.interactive.content.pptx)
[1129_ML.NET_Img.Clas_NVDA_Fixed.focus.is.not.going.to.extra.content.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/6042287/1129_ML.NET_Img.Clas_NVDA_Fixed.focus.is.not.going.to.extra.content.pptx)

Verified the bug in the latest build and the issue is fixed.
Hence closing the bug
Please find the attachment for reference
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.9.0 Preview 4.0

**Comment by poojapatil36** [comment]
Github tag: #Closed;
